pkg/resource/function/hooks.go

```diff
@@ -497,41 +497,53 @@ func (rm *resourceManager) updateFunctionEventInvokeConfig(
 ) error {
 	var err error
 	rlog := ackrtlog.FromContext(ctx)
-	exit := rlog.Trace("rm.updateFunctionEventInvokeConfig")
+	exit := rlog.Trace("rm.syncEventInvokeConfig")
```

Comment:
I imagine you changed this because now we don't only "update", but we can delete too.. But I wonder if it's confusing in the logs to see a trace for "syncEventInvokeConfig`, but to not actually have a function called like that. (I don't know what's the best name between the sync and update, but I think we should be consistent between the trace logs and the function name)

Reply:
Yeah the reason behind "sync" was the same as you mentioned. I have changed the function name to "syncEventInvokeConfig" as even in other PR, we go with "syncEventInvokeConfig" for the function name.